.github/workflows/docker.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Build docker image
         run: docker build . -t doctr-tf-py3.8-slim
       - name: Run docker container
-        run: docker run doctr-tf-py3.8-slim python -c 'import doctr'
+        run: docker run doctr-tf-py3.8-slim python3 -c 'import doctr'
```

Comment:
`python` does not exist. It's possible to add a symlink `python -> python3` but it may cause trouble in the Docker Container

Reply:
Mh personally on every linux machine i work i use always `python3` so from my side that's totally fine 👍🏼